### DISCUSSION TITLE
Unbound LXC Container

### DISCUSSION THREAD

**Discussion by wimb0** [community-scripts/ProxmoxVE#530] [Request script]
Hi,

Thanks for all those helper scripts. They are excellent!

I could not find a separate Unbound LXC Container, so I created one (based on your scripts).
(Only the one integrated in Pi-Hole, but I do not use that)
See: https://github.com/wimb0/unbound-lxc

Maybe it's an idea to add this to the helper scripts?

**Comment by Mellowlynx** [comment]
You are welcome to add it as a Pull request so it can be added.

**Comment by Mellowlynx** [comment]
You are welcome to add it as a Pull request so it can be added.

**Comment by wimb0** [comment]
created pull request: https://github.com/community-scripts/ProxmoxVE/pull/547
